ppr-ui/src/components/mhrTransfers/SupportingDocuments.vue

```diff
@@ -151,6 +153,10 @@ export default defineComponent({
   display: flex;
   flex-direction: column;
 
+  :deep(.non-clickable-label .v-label) {
+    pointer-events: none;
```

Comment:
This prevents clicking on the label of the Supporting Document radio buttons. When clicking on label, and not on the radio button itself, was causing issues with wrong selections, which I could not figure out.